pkgs/by-name/vm/vmlinux-to-elf/package.nix

```diff
@@ -0,0 +1,32 @@
+{
+  lib,
+  fetchFromGitHub,
+  python3,
+}:
+python3.pkgs.buildPythonPackage {
```

Comment:
buildPythonPackage is for python modules/dependencies under pkgs/development/python-modules, not standalone applications (see Nixpkgs manual for the difference). Use buildPythonApplication instead